src/stream.rs

```diff
@@ -84,6 +84,7 @@ impl Stream {
         Stream { c_stream }
     }
 
+    // TODO: document how this is different from `Default::default()`
```

Comment:
We could name this `default_stream_for_device()`?

Reply:
Sounds good. I will make that change in this PR

Reply:
Renamed to `default_stream_on_device`